google-cloud-speech/acceptance/speech/async_test.rb

```diff
@@ -166,4 +166,21 @@
     results.first.confidence.must_be_close_to 0.9, 0.1
     results.first.alternatives.must_be :empty?
   end
+
+  it "recognizes audio with words" do
+    op = speech.process filepath, encoding: :linear16, language: "en-US", sample_rate: 16000, words: true
```

Comment:
This should be called "enable_word_time_offsets" for consistency with other APIs and the upcoming auto-generated client libraries, and for future-proofness.

word_time_offsets are enabled individually, and we will add other word level features in the future which will have their own flags, so calling this "words" is too generally.